MoreLinq/Rank.cs

```diff
@@ -32,7 +32,7 @@ public static partial class MoreEnumerable
 
         public static IEnumerable<int> Rank<TSource>(this IEnumerable<TSource> source)
         {
-            return source.RankBy(x => x);
+            return source.RankBy(Identity<TSource>);
```

Comment:
Type argument is redundant:

```suggestion
            return source.RankBy(Identity);
```